src/app/domain/entities/champion/champion-entity.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { ChampionEntity } from './champion-entity';
+
+const sut = (): ChampionEntity => {
+  return new ChampionEntity({
+    name: 'Aatrox',
+    key: '266',
+    tags: ['Fighter', 'Tank'],
+    stats: {
+      hp: 650,
+      armor: 38,
+      attackdamage: 60,
+      attackspeed: 0.651,
+    },
+  });
+};
```

Comment:
I think your sut should receive an object, not mock directly in the constructor.
You can use this object to check the equalities of the attributes, and if you change a value from object, you won't need to change the test case.